content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -108,32 +115,40 @@ cluster for mesh expansion, run the following commands on a machine with cluster
           -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
     {{< /text >}}
 
-### Setting up the machines
+### Setting up VM machines
```

Comment:
s/VM machines/the VMs

Reply:
I feel definite articles are not needed here for plural forms. Not quite confident since not a native speaker, @justinburke thoughts?

Reply:
I meant "VM machines" is literally "Virtual Machine machines". You can remove the second "machine".

Reply:
Maybe we can resolve this by saying "Setting up the VM" since this guide explicitly shows the user how to setup a single VM, not a collection.

Reply:
Done.